### PR TITLE
Add badge with emancipation checklist summary

### LINKED ISSUE #2470: Add Emancipation Checklist Summary to Emancipation Button
**What type(s) of user does this feature affect?**
 - volunteers
 - supervisors
 - admins

**Description**
Currently there is a plain button to link to the appropriate emancipation checklist from a case details page. We would like this button to have a fraction on it representing how complete the emancipation checklist is i.e. 0/18 (where 0 is the number of top level emancipation items they've entered results for out of 18 total).

**Screenshots of current behavior, if any**  
Emancipation button

_QA Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin

password for all users: 123456

### PR DESCRIPTION
**Pull request by r-sierra**
### What github issue is this PR for, if any?
Resolves #2470

### What changed, and why?
Added a summary to the Emancipation button showing the amount of emancipation categories done

### How will this affect user permissions?
No user permission affected

### Screenshots please :)
Captura de pantalla de 2021-08-31 14-25-17

### CODE REVIEW

File: app/views/casa_cases/show.html.erb [hunk 1]
```diff
@@ -95,7 +98,7 @@
 
     <div>
       <h6><strong><%= t(".assigned_volunteers") %>:</strong></h6>
-      <% policy_scope(@casa_case.assigned_volunteers).each_with_index do |volunteer, index| %>
+      <% @casa_case.assigned_volunteers.each do |volunteer| %>
```
Comment (compwron) [inline app/views/casa_cases/show.html.erb:101]: bring back policy :)

File: app/decorators/casa_case_decorator.rb [hunk 2]
```diff
@@ -79,4 +79,8 @@ def unsuccessful_contacts_this_week_before(date)
     this_week_before_date = Date.today - 7.days..date
     object.case_contacts.where(occurred_at: this_week_before_date).where(contact_made: false).count
   end
+
+  def emancipation_checklist_count
```
Comment (compwron) [inline app/decorators/casa_case_decorator.rb:83]: add a test please :)

**Review by compwron — CHANGES_REQUESTED** [review]

File: app/views/casa_cases/show.html.erb (hunk 1 above)
Comment (r-sierra) [reply app/views/casa_cases/show.html.erb:101]: that wasn't supposed to be commited, will undo

**Review by compwron — APPROVED** [review]
:)

**Pull request merged**